### PR TITLE
Final changes

### PR DESCRIPTION
**Pull request by Tharun1211**

### CODE REVIEW

**Comment by neetisharma019** [comment]
Please resolve conflict

**Comment by neetisharma019** [comment]
Merging final changes, looks good

File: story.txt [hunk 1]
```diff
@@ -6,8 +6,8 @@ I am writing to you from a [2] spaceship in a [3] galaxy. I found myself here on
...
 [10] there is a pool full of water. I fall asleep each night on a beautiful bed of
 wood and dream of singing slowly. It feels as though I have lived here for
-[16] [17]. I hope one day you can visit, although the only way to get here now is
-[18] on a [19] [20]!!
+420 galactic years. I hope one day you can visit, although the only way to get here now is
+drowning on a tripping tricycle!!
 
 Best wishes,
-Capt. Crunch
+Oppenheimer
```
Comment (neetisharma019) [inline story.txt:13]: Conflict has been resolved and changes have been reviewed.

**Review by Chaitanya1507 — COMMENTED** [review]
all good

**Pull request merged**